package.json

```diff
@@ -37,6 +37,7 @@
     "db:migrate-down-all": "node ./src/server/database/migrate.babel.js down --all",
     "debug:stripe": "node ./src/server/billing/debug.babel.js",
     "test": "jest --bail --coverage --ci --runInBand && codecov",
+    "test:dev": "jest --runInBand --watch",
```

Comment:
Here's what `jest --help` has to say:

```
  --runInBand, -i                 Run all tests serially in the current process
                                  (rather than creating a worker pool of child
                                  processes that run tests). This is sometimes
                                  useful for debugging, but such use cases are
                                  pretty rare.                         [boolean]
```

Seems like it may make attaching a debugger easier. Then again, the `--watch` argument doesn't work for me here; it quits after a test run. Not sure if this has to do with out codebase or my laptop. So I'd love to rework this script to actually do what it's supposed to ;)

While discussing, why are we passing `--runInBand` for the canonical `test` script?

Reply:
jest steals too much memory in circleCI (it's a known bug). the workaround is to not spawn children.

Reply:
Aw dang, that's sad. Thanks for the clarification!